platform/ios/src/MGLMapView.mm

```diff
@@ -2984,6 +2984,15 @@ - (MGLAnnotationView *)annotationViewForAnnotation:(id<MGLAnnotation>)annotation
     return annotationView;
 }
 
+- (nullable MGLAnnotationView *)viewForAnnotation:(id<MGLAnnotation>)annotation
+{
+    MGLAnnotationTag annotationTag = [self annotationTagForAnnotation:annotation];
+    if (annotationTag == MGLAnnotationTagNotFound) return nil;
```

Comment:
Eventually, if the developer passes in `userLocation`, this method should return the MGLUserLocationAnnotationView, but this is fine for now because MGLUserLocationAnnotationView doesn't subclass MGLAnnotationView yet.